### PR TITLE
Added a document for running argo workflow on K3s. Fixes: #63

### PR DESCRIPTION
**Pull request by yashgangwar7558**
@patchandpray Added a blog/document that describes steps in detail to set up K3s locally and run agro workflows. Please review it and give some suggestions.

Addresses issue #63 and #3

### CODE REVIEW

**Comment by AmitKarnam** [comment]
Great blogs @yashgangwar7558 , very informative reads.

File: resources/How_do_I_run_an_Argo_Workflow_on_K3S.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+   kube-node-lease   Active   8h
+   argo              Active   8h
+   ```
+   
+**:white_check_mark: Step 2: Install Argo Workflows**
+
+   To get started quickly, we can use the quick start manifest which will install Argo Workflow as well as some commonly used components
+   
+   ```sh
+   kubectl apply -n argo -f https://raw.githubusercontent.com/argoproj/argo-workflows/master/manifests/quick-start-postgres.yaml
```
Comment (shreemaan-abhishek) [inline resources/How_do_I_run_an_Argo_Workflow_on_K3S.md:107]: I see that this is for Postgres, could you please replace it with MongoDB as we are going to use MongoDB in our project? What are your thoughts @patchandpray @AbhijithGanesh

File: resources/How_do_I_run_an_Argo_Workflow_on_K3S.md [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+   You can see your workflow named `hello-world-xpc6d` has started running. Wait till it has finished executing. You can also see your workflow on the dashboard
+    
+**:white_check_mark: Step 2: Getting logs of our workflow**
+
+   Replace `@latest` with the name of your workflow. For eg. in my case its `hello-world-xpc6d`
+
+   ```sh
+   argo list -n argo
+   argo get -n argo @latest
+   argo logs -n argo @latest
```
Comment (shreemaan-abhishek) [inline resources/How_do_I_run_an_Argo_Workflow_on_K3S.md:194]: You said,
> Replace `@latest` with the name of your workflow. For eg. in my case its `hello-world-xpc6d`

But you're still using the @latest tag in this command. Could you please clarify?

**Review by shreemaan-abhishek — COMMENTED** [review]
I have added my points, other than that LGTM!

File: resources/How_do_I_run_an_Argo_Workflow_on_K3S.md (hunk 1 above)
Comment (AmitKarnam) [reply resources/How_do_I_run_an_Argo_Workflow_on_K3S.md:107]: I think the database to use is still in consideration and the discussion is still going on in the issue #69

File: resources/How_do_I_run_an_Argo_Workflow_on_K3S.md (hunk 2 above)
Comment (yashgangwar7558) [reply resources/How_do_I_run_an_Argo_Workflow_on_K3S.md:194]: Here @latest addresses to the recent or the last submitted workflow. You can also replace it with the name of the workflow.

Thanks for reviewing!! I will fix this and make it more clear in the document.

File: resources/How_do_I_run_an_Argo_Workflow_on_K3S.md (hunk 1 above)
Comment (yashgangwar7558) [reply resources/How_do_I_run_an_Argo_Workflow_on_K3S.md:107]: Actually, this was the quick start manifest file to just install Argo workflows and it was already provided by Argo. It's quite complicated to understand and create our own manifest file, so I guess for beginners it's the best way, to just get started with workflows.

**Comment by yashgangwar7558** [comment]
@shreemaan-abhishek I have fixed the requested changes related to **@latest**. I hope now it's more clear?

**Comment by shreemaan-abhishek** [comment]
Yes it is!

**Review by shreemaan-abhishek — APPROVED** [review]
LGTM!

**Comment by patchandpray** [comment]
@yashgangwar7558 one comment. Can you change the name of the file to all lowercase_underscores

**Comment by patchandpray** [comment]
I'm going through the steps this evening to see if I can run a workflow :D, will get back to you.

**Comment by yashgangwar7558** [comment]
@patchandpray Thanks for reviewing. I have made the requested changes.

**Comment by shreemaan-abhishek** [comment]
Turns out this also fixes #3. Edited the main issue comment.

**Review by patchandpray — APPROVED** [review]

**Pull request merged**